humilis_sam/resources.yaml.j2

```diff
@@ -16,20 +25,21 @@ resources:
                 {% endfor %}
     {% endif %}
 
+    {% set default_description="SAM app" %}
     {% for f in meta_functions %}
     {{f.name}}:
         Type: AWS::Serverless::Function
         Properties:
             Handler: handler.lambda_handler
             Runtime: {{runtime}}
             CodeUri: {{lambda_function}}
-            Description: {{func_description}}
+            Description: "{{f.description or default_description}} ({{_env.name}}-{{_env.stage}})"
             MemorySize: {{f.memory_size}}
             Timeout: {{f.timeout}}
             {% if iam_actions %}
             Role:
                 "Fn::GetAtt":
-                    - LambdaRole{{_layer.name}}{{_env.stage}}
```

Comment:
I prefer this shorter name and both layer name and stage are already part of the name of the CF stack associated to the humilis layer.